### ISSUE TITLE
Usage invoice item description does not use the pretty name

### ISSUE THREAD

**Issue by hendryluk** [killbill/killbill#1183]
This line:
https://github.com/killbill/killbill/blob/e30cdaef205ac51340217679e70dd9d17eb4ae6b/invoice/src/main/java/org/killbill/billing/invoice/model/UsageInvoiceItem.java#L59

Should that be `MoreObjects.firstNotNull(description, prettyUsageName)`?
Currently it's showing the ugly unique-key, which is unhelpful for the customers, and also inconsistent with other implementations of description that's normally in plain english, e.g. https://github.com/killbill/killbill/blob/e30cdaef205ac51340217679e70dd9d17eb4ae6b/invoice/src/main/java/org/killbill/billing/invoice/model/FixedPriceInvoiceItem.java#L61

**Comment by sbrossie** [comment]
@hendryluk I think that makes sense, but please report issues on the mailing list.
